src/test/java/com/redhat/cloud/notifications/routers/ApplicationServiceTest.java

```diff
@@ -147,4 +145,53 @@ void testGetApplicationsRequiresBundleName() {
                 .statusCode(500);
 
     }
+
+    @Test
+    void testGetApplicationsReturnApplications() {
```

Comment:
Would it be possible to improve `testPoliciesApplicationAddingAndDeletion()` instead of adding this test? It looks like they have a lot in common but maybe I missed a significant difference.

Reply:
They share a very similar setup, but the other is testing adding/deletion. This one focus on verifying that the contents of the API call is indeed an Application (in this case, I added 10 applications). IMO is OK to repeat some code to have a more focused test when possible.

If needed, the supporting setup code could be refactored in some helper method to be more concise.

Reply:
Makes sense, let's keep it that way.

Reply:
I can't resolve my own comment because I don't have the required rights on the repo, but this is resolved 😄